### ISSUE TITLE
Gauges show up twice in React 18 Strictmode

### ISSUE THREAD

**Issue by hunterEdward98** [telerik/kendo-react#1363]
## I'm submitting a...
* Bug report 

## Current behavior
all kendo-react-gauges components render twice in React 18 Strictmode

## Expected behavior
Renders once

### run the command
`npx create-react-app kendo-gauge-duplicates --template typescript`
followed by 
`cd kendo-gauge-duplicates`
then
`npm i @progress/kendo-react-gauges`

#### Overwrite everything with the following code
```
import "./App.css";
import { ArcGauge } from "@progress/kendo-react-gauges";

const colors = [
	{
		from: 0,
		to: 20,
		color: "#F00",
	},
];
function App({ value }: { value: number }) {
	const arcCenterRenderer = (value: number, color: string) => {
		return <h3 style={{ color: color }}>{value}%</h3>;
	};
	return (
		<div className="App">
			<header className="App-header">
				<ArcGauge
					colors={colors}
					value={value}
					arcCenterRender={arcCenterRenderer}
				/>
			</header>
		</div>
	);
}

export default App;
```

### in index.tsx
add a `value={20}` to the App object 

## What is the motivation or use case for changing the behavior?
to make the component render appropriately in strict mode

## Environment

Package versions:

```
kendo-gauges-broken@0.1.0 C:\Software\Kendo\kendo-gauges-broken
├── @progress/kendo-react-gauges@5.5.0
├── @testing-library/jest-dom@5.16.4
├── @testing-library/react@13.3.0
├── @testing-library/user-event@13.5.0
├── @types/jest@27.5.2
├── @types/node@16.11.43
├── @types/react-dom@18.0.6
├── @types/react@18.0.15
├── react-dom@18.2.0
├── react-scripts@5.0.1
├── react@18.2.0
├── typescript@4.7.4
└── web-vitals@2.1.4
```

Browser:
- Chrome (desktop) version 103.0.5060.114
System:
- TypeScript version: 4.7.4
- Node version: 18.14.0
- Npm version: 8.12.1
- Platform:  Windows 10

**Comment by edouardmisset** [comment]
Same here

**Comment by nstoychev** [comment]
Fixed in our development channel version 5.5.1-dev.202207141327.

Example: https://stackblitz.com/edit/react-mj12dg?file=app%2Fmain.jsx.
